### ISSUE TITLE
[Accessibility issue] Click on subscribe button with empty Name and email does not read out as value missing from both text boxes

### ISSUE THREAD

**Issue by harshamulpuri2913** [ampproject/amphtml#3880]
## Short description of your issue: within iOS device (IPAD / IPOD), observe "value missing" is not read out by voice over from Name & email text boxes when clicked on subscribe button. 

Test URL: https://amp-mk-1.herokuapp.com/examples.build/forms.amp.max.html

Pre-requisite: Open IPAD--> Settings --> General --> Accessibility --> Voiceover , Set to ON
## How do we reproduce the issue?
1. Logon to IPAD and open the safari browser
2. Open test URL provided above
3. Under "Subscribe to our weekly newsletter (non-xhr)", please click on submit button without entering Name OR email values

Actual results: 
1. "Value missing" tooltip gets displayed within the Name text box upon click on subscribe button
2.  Message "There are some input that needs fixing. Please fix them" is displayed but screen reader does not read it out.
3. Focus is set on Name text box and voiceover reads as "your name text field is editing character mode required insertion point at start"

Expected result: 
1. With both Name & email edit boxes set to NULL / No value, upon click on Subscribe button, focus should be set on Name edit box and voiceover should read out as "Value missing within Name text box. this is a mandatory or required field".
2. Similarly for email text box. 

## What browsers are affected? Safari , Chrome

All browsers? Some specific browser? What device type?
Device details: IPAD 
iOS version: 9.3.2 (13F69)

Attachments: 
![img_0043](https://cloud.githubusercontent.com/assets/20247457/16532555/89382da8-3f87-11e6-92f6-260748677a9b.PNG)

**Comment by mkhatib** [comment]
Thanks @harshamulpuri2913 for filing this! Definitely wanted to address this and thanks for the detailed description.

**Comment by mkhatib** [comment]
I am working on fixing this. Struggling to really get the `VoiceOver` to trigger it everytime, it seems that VO is triggering sometimes and others not. 

There's also seem to be a problem with browsers provided validation bubbles not being accessible either.
